composer.json

```diff
@@ -23,7 +23,8 @@
 		"php": ">=5.5.0",
 		"wikibase/data-model": "~6.0|~5.0|~4.2",
 		"data-values/data-values": "~0.1|~1.0",
-		"diff/diff": "~2.0|~1.0"
+		"diff/diff": "~2.0|~1.0",
```

Comment:
True. Did it in https://github.com/wmde/WikibaseDataModelServices/pull/146, we still need to get DM 6.2 though